src/Loading.css

```
@@ -0,0 +1,44 @@
+.loading {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.loading-circle {
+  width: 54px;
+  height: 54px;
+  animation: loading-spin-ani 0.5s infinite linear;
+}
+
+.loading-circle circle {
+  stroke: #000;
+  stroke-width: 4;
+  fill: none;
+  stroke-dasharray: 157;
+  stroke-dashoffset: 157;
+  animation: loading-circle-ani 1s infinite;
+}
+
+@keyframes loading-circle-ani {
+  0% {
+      stroke-dashoffset: 0;
+  }
+  65% {
+      stroke-dashoffset: -147;
+  }
+  100% {
+      stroke-dashoffset: -157;
+  }
+}
+
+@keyframes loading-spin-ani {
+  to {
+      transform: rotate(360deg);
+  }
+}
```

```suggestion
}
// Some new code
```

코드 마지막 줄에는 빈 줄이 하나가 추가되어야 합니다. 그렇지 않으면 위와 같이 GitHub에서 표시가 됩니다.

빈 줄을 추가해야하는 이유는 GitHub의 컨벤션 때문에 그렇습니다. GitHub은 변경사항을 표시할 때 변경전 부분과 변경 후 를 모두 보여줍니다. 

```js
console.log('Hello');
```

만약 위와 같이 빈 줄이 없는 코드에서 아래에 코드를 추가하게 되면 Git은 다음과 같이 표시합니다.

```js
- console.log('Hello');
+ console.log('Hello');
+ console.log('world!');
```

저는 단순히 아래 한 줄만 추가했는데 왜 세줄이 변경되었다고 나오는 걸까요? 그 이유는 줄바꿈 문자 \n가 들어갔기 때문입니다. 


```js
- console.log('Hello');
+ console.log('Hello');\n
+ console.log('world');
```

만약 미리 빈 줄을 하나 추가한 상태에서 아래에 새로운 코드를 작성할 경우에는 다음과 같이 보여집니다.

```js
+ console.log('world');
```

이미 \n이 추가되어 있기 때문이죠. 따라서 코드 마지막 줄에 빈 줄 하나를 항상 추가해주는게 좋습니다.